pgml-cms/docs/api/client-sdk/README.md

```diff
@@ -99,7 +99,7 @@ async fn main() -> Result<(), Error> {
 {% endtab %}
 
 {% tab title="C" %}
-```c
+```cpp
```

Comment:
You may need to install the cpp syntax highligher extension for our text editor.

Reply:
I did!

Reply:
There was two prs, this was meant to be in the first one but I accidentally merged it, so had to add this one after.

https://github.com/postgresml/postgresml/pull/1506

Reply:
Nice!